### DISCUSSION TITLE
MCDU fpl problems.

### DISCUSSION THREAD

**Discussion by bmelen** [flybywiresim/aircraft#3668] [Help]
When I try to input the flight plan from Simbrief to MCDU using the following steps. After filling in all the pages except the departure and arrival details. I go to departure airport and chose a runway and a SID and when INSERT. After that I can´t go back into the flight plan again. Nothing happens when I  push the fpl button. Happens in both stable version and dev mod.

**Comment by MapleNAV-TCASFAULT** [comment]
Make sure that you don't have any conflicting mods in your community folder
